static/src/javascripts/projects/common/modules/commercial/adblock-messages.js

```diff
@@ -31,23 +31,19 @@ define([
         return config.switches.adblock;
     }
 
-    function noAdblockMsg() {
```

Comment:
👍  for getting rid of this quite unintelligible function! Are we sure it's not called anywhere other than tests though?